api/task_get_test.go

```diff
@@ -69,3 +69,81 @@ func TestTaskLifeCycleHandler_GetTaskByName(t *testing.T) {
 		})
 	}
 }
+
+func TestTaskLifeCycleHandler_GetTasks(t *testing.T) {
```

Comment:
nit: This corresponds to GetAllTasks rather than GetTasks, but also maybe we could just call that method GetTasks?

Reply:
yeah, I renamed the method to `GetAllTasks` from `GetTasks` and it didn't catch everything, I'll fix this and others if I see them